src/screens/LocationDetailsScreen.tsx

```diff
@@ -1,19 +1,364 @@
+                "",
+                "3pm",
+                "",
+                "",
+                "",
+                "",
+                "",
+                "",
+                "",
+                "11pm",
+              ]}
+              style={{
+                tickLabels: { fill: theme.styles.placeholderText.color },
+                axis: { stroke: "none" },
+              }}
+            />
+            <VictoryBar
+              style={{
+                data: {
+                  fill: "#1677FF",
+                  opacity: ({ datum }) =>
+                    datum.quarter === poiData.currentHour ? 1 : 0.5,
+                },
+              }}
+              cornerRadius={{ top: 3, bottom: 3 }}
+              data={chartData}
+              x="quarter"
+              y="earnings"
+            />
+          </VictoryChart>
+        </View>
+      </ScrollView>
+      <View style={styles.waitTimeContainer}>
+        <View>
+          <StyledText style={styles.waitTimeText}>Wait Time</StyledText>
+          <StyledText style={styles.detailsText}>
+            {"Current time is " +
+              poiData.waitTime +
+              (poiData.waitTime === 1 ? " min" : " mins") +
+              " • " +
+              renderLastUpdated(poiData.lastUpdated)}
+          </StyledText>
+        </View>
+        <View style={styles.confirmWaitTimeButtonContainer}>
+          <Button
+            style={[theme.styles.confirmWaitTimeButton, styles.waitTimeButtons]}
+            onPress={confirmNewWaitTime}
+          >
+            <StyledText
+              style={[
+                theme.styles.primaryColor,
+                styles.confirnWaitTimeButtonText,
+              ]}
+            >
+              {"Confirm " +
+                poiData.waitTime +
+                (poiData.waitTime === 1 ? " min" : " mins")}
+            </StyledText>
+          </Button>
+        </View>
+        <View style={styles.newWaitTimeContainer}>
+          <View style={styles.newWaitTimeButtonsContainer}>
+            <Button
+              size="large"
+              style={[
+                theme.styles.newWaitTimePlusMinusButton,
+                styles.waitTimeButtons,
+                styles.newWaitTimePlusMinusButton,
+                styles.newWaitTimeMinusButton,
+              ]}
+              disabled={newWaitTime === 0}
+              onPress={() => setNewWaitTime((prevCount) => prevCount - 1)}
+            >
+              <AntDesign
+                name="minus"
+                size={18}
+                color={theme.styles.primaryColor.color}
+              />
+            </Button>
+            <View
+              style={[
+                theme.styles.newWaitTimeTextContainer,
+                styles.newWaitTimeTextContainer,
+              ]}
+            >
+              <StyledText style={[theme.styles.text, styles.newWaitTimeText]}>
+                {newWaitTime + (newWaitTime === 1 ? " min" : " mins")}
+              </StyledText>
```

Comment:
Could you fix this width so that the container doesn't expand when you go from 9 minutes to 10?